api/src/services/opportunity_attachments/attachment_util.py

```diff
@@ -27,7 +27,7 @@ def get_s3_attachment_path(
     return file_util.join(
         base_path,
         "opportunities",
-        str(opportunity.opportunity_id),
+        str(opportunity.legacy_opportunity_id),
```

Comment:
For this function, it might be better to just use the UUIDs instead of the integers. Only reason to keep the integers is if we really want to come up with a way to not re-import the attachments (annoying to do, but pretty whatever).